src/nbl/disable_3rdparty_warnings.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _NBL_DISABLE_3RDPARTY_WARNINGS_
```

Comment:
okay let's remove this header and move the macro into [`BuildConfigOptions.h.in` here](https://github.com/Devsh-Graphics-Programming/Nabla/blob/10d4e038d14938c2ae974e156839ef9060ed5763/include/nbl/config/BuildConfigOptions.h.in#L108)